tests/Ltest-mem-validate.c

```diff
@@ -49,6 +49,12 @@ void do_backtrace (void)
 {
   void* buffer[1024];
   int size = 1024;
```

Comment:
nitpick, while you are touching this function could you delete the three unused variables
```sh
Ltest-mem-validate.c: In function ‘do_backtrace’:
Ltest-mem-validate.c:51:7: warning: unused variable ‘size’ [-Wunused-variable]
   51 |   int size = 1024;
      |       ^~~~
Ltest-mem-validate.c:50:9: warning: unused variable ‘buffer’ [-Wunused-variable]
   50 |   void* buffer[1024];
      |         ^~~~~~
Ltest-mem-validate.c: In function ‘main’:
Ltest-mem-validate.c:133:15: warning: unused variable ‘ret’ [-Wunused-variable]
  133 |           int ret = close(i);
      |               ^~~
```